lookup.go

```diff
@@ -54,6 +54,21 @@ func lookup(name string, context ...interface{}) (interface{}, bool) {
 				return out.Interface(), truth(out)
 			}
 
+			typ := reflectValue.Type()
+			for i := 0; i < typ.NumField(); i++ {
+				f := typ.Field(i)
+				if f.PkgPath != "" {
```

Comment:
nit: perhaps add a comment saying that this is skipping unexported fields.


Also, unrelated to your change, but seems the code above is dangerous:
```
field := reflectValue.FieldByName(name)
if field.IsValid() {
  return field.Interface(), truth(field)
}
```
When the field is an unexported field, this code will panic. I'd prefer to catch it and returns `nil, false` instead.